Formula/gambit-scheme.rb

```diff
@@ -0,0 +1,127 @@
+class GambitScheme < Formula
+  desc "Gambit Scheme"
+  homepage "http://gambitscheme.org"
+  url "https://github.com/gambit/gambit.git", :tag => "v4.9.0"
+
+  depends_on "gcc@6"
+
+  def install
+    args = %W[--prefix=#{prefix}]
+
+    ENV["CC"] = Formula["gcc@6"].bin/"gcc-6"
+
+    if build.with? "disable-ssl-verification"
```

Comment:
No options. Provide a "best use case" version that will have most features and be useful for as many users as possible.